aggregator/fetchers/baseFetcher.go

```diff
@@ -0,0 +1,16 @@
+package fetchers
+
+import "strings"
+
+type baseFetcher struct{}
+
+func (b *baseFetcher) updateQuoteIfNeeded(quote *string, fetcherName string) {
```

Comment:
why not make this function properly return a string and name it something like `normalizeQuoteName` or something?
It will be something like `normalizeQuoteName(quote string, fetcherName string) string`
The usage will be something like:
```GO
//.....
quote = b.normalizeQuoteName(quote, binanceName)
//.....
```

The main reason is this: we should try at maximum to avoid changing the input parameters of a function. Otherwise it will be harder to understand and debug by someone else.